lib/auth/index.ts

```diff
@@ -52,14 +55,14 @@ export const login = async () => {
 	// from mixed content warnings (as the target of a form in the result page)
 	const callbackUrl = `http://127.0.0.1:${options.port}${options.path}`;
 	const loginUrl = await utils.getDashboardLoginURL(callbackUrl);
+	console.info(`Opening web browser for URL:\n${loginUrl}`);
```

Comment:
This changes the current output, from:
```
$ ./bin/balena login
Logging in to balena-cloud.com
? How would you like to login? Web authorization (recommended)
Connecting to the web dashboard
```
To:
```
$ ./bin/balena login
Logging in to balena-cloud.com
? How would you like to login? Web authorization (recommended)
Opening web browser for URL:
https://dashboard.balena-cloud.com/login/cli/http%253A%252F%252F127.0.0.1%253A8989%252Fauth
```
If you have any reservations about this change, please say so! It partially addresses issue #1683, which requires a URL to be printed so that web authorization can be performed on a different machine than the one the CLI is running on. _"Partially"_ because, as the moment, the printed URL uses a `127.0.0.1` redirect IP address, which is no good for remote authorisation. Still, the user could copy-and-paste the URL and manually replace the redirect IP address on the URL.